ext/B/t/showlex.t

```diff
@@ -49,10 +48,6 @@ for $newlex ('', '-newlex') {
 
     print $out if $verbose;
 
-SKIP: {
-    skip "no perlio in this build", 5
-    unless $Config::Config{useperlio};
-
     our $buf = 'arb startval';
     my $ak = B::Showlex::walk_output (\$buf);
 
```

Comment:
Can we de-indent some lines here because we're no longer using a `SKIP:{}` block?

Reply:
Not that one - GitHub didn't do you any favours with context. The two diff hunks look like this:

```
@@ -49,10 +48,6 @@ for $newlex ('', '-newlex') {

     print $out if $verbose;

-SKIP: {
-    skip "no perlio in this build", 5
-    unless $Config::Config{useperlio};
-
     our $buf = 'arb startval';
     my $ak = B::Showlex::walk_output (\$buf);

@@ -102,4 +97,3 @@ SKIP: {
     $walker->();

 }
-}

```

ie when the `SKIP` block was added, the lines within it weren't indented.

Reply:
Yes, I have confirmed that.